### PR TITLE
GonnaGetBetter

### PR DESCRIPTION
**Pull request by gonnagetbetter**
Added mazeGenerator and made some fixes.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Maze from './classes/mazeGenerator.js';
+
+const maze = new Maze(10, 10);
+
+maze.generate();
+
+console.log(maze.maze);
```
Comment (Alzex) [inline src/index.js:7]: Make `generate` function return value of maze

File: src/classes/mazeGenerator.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+export default class Maze {
+  constructor(height, width) {
+    this.height = height;
+    this.width = width;
+
```
Comment (Alzex) [inline src/classes/mazeGenerator.js:5]: Add start and exit for the maze

```js
// Create entrance and exit
maze[1][0] = false;
maze[height*2-1][width*2] = false;
```

**Review by Alzex — CHANGES_REQUESTED** [review]

File: src/classes/mazeGenerator.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    this.maze = Array(height * 2 + 1)
```
Comment (Alzex) [inline src/classes/mazeGenerator.js:6]: rename `maze` field to something different
for example: `values`

**Comment by gonnagetbetter** [comment]
Fixed these issues

**Review by Alzex — APPROVED** [review]

**Pull request merged**